Algorithm/QCAlgorithm.Python.cs

```diff
@@ -370,6 +370,21 @@ public void RegisterIndicator(Symbol symbol, PyObject indicator, IDataConsolidat
             };
         }
 
+        /// <summary>
+        /// Plot a chart using string series name, with value.
+        /// </summary>
+        /// <param name="series">Name of the plot series</param>
+        /// <param name="pyObject">PyObject with the value to plot</param>
+        /// <seealso cref="Plot(string,decimal)"/>
+        public void Plot(string series, PyObject pyObject)
+        {
+            using (Py.GIL())
+            {
+                var value = ((dynamic)pyObject).Current.Value;
```

Comment:
Is it possible to to check `pyObject` for implemented interfaces to confirm it implements `IIndicator<T>`?

Concern is non-indicator `PyObject`s getting in here by accident, would like to minimally log for us and message the user if `pyObject` isn't an indicator.

What about a decimal value in python, will that bind to this method as a `PyObject`?
``` python
self.sma = SMA("SPY", 10)
self.value = sma.Current.Value
Plot("SPY-Price", self.value);
```